dim_checks/moz-fx-data-shared-prod/telemetry/unified_metrics/dim_checks.yaml

```diff
@@ -0,0 +1,95 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  alerts_enabled:
+    enabled: true
+    notify:
+      channels:
+        - dim_test_run
+  tier: tier_1
+  dim_tests:
+    - type: table_row_count
+      params:
+        condition: ">= 1000"
+    - type: not_null
+      params:
+        columns:
+          - submission_date
+          - client_id
+          - sample_id
+          - activity_segment
+          - normalized_app_name
+          - normalized_channel
+          - country
+          - city
+          - days_seen_bits
+          - days_since_first_seen
+          - days_since_seen
+          - device_model
+          - is_new_profile
+          - first_seen_date
+          - locale
+          - normalized_os
+          - durations
+          - normalized_os_version
+          - app_version
+          - os_version_major
+          - os_version_minor
+          - os_version_patch
+    - type: column_length
```

Comment:
The client_id also seems to have a specific length. I'd double check probably in the data channel.

Reply:
Looks like it's `36` characters long. I'll double check if this assumption is always correct.

Reply:
Added the check.